solutions/USACO dec15-breed-counting.mdx

```diff
@@ -13,37 +13,43 @@ author: Óscar Garries (C++), Owen Wang (Java)
 ## Implementation
 
 ```cpp
-#include <bits/stdc++.h>
+#include <iostream>
 
-using namespace std;
+using std::cin;
```

Comment:
i don't think it's necessary to rewrite others' solutions

Reply:
that doesn't mean it's bad